data/common.yaml

```diff
@@ -36,6 +36,8 @@ prosody::modules_base:
 prosody::modules_disabled: []
 prosody::package_ensure: present
 prosody::package_name: prosody
+prosody::manage_repository: true
```

Comment:
I think the default should be false, otherwise we accidentally upgrade prosody?

Reply:
fixed